server/notification-providers/teams.js

```diff
@@ -21,18 +21,32 @@ class Teams extends NotificationProvider {
     };
 
     /**
-     * Select theme color to use based on status
+     * Select the style to use based on status
      * @param {const} status The status constant
-     * @returns {string} Selected color in hex RGB format
+     * @returns {string} Selected style for adaptive cards
      */
-    _getThemeColor = (status) => {
+    _getStyle = (status) => {
         if (status === DOWN) {
-            return "ff0000";
+            return "attention";
         }
         if (status === UP) {
-            return "00e804";
+            return "good";
+        }
+        return "emphasis";
+    };
+
+    /**
+     * Format an URL in the markdown format
+     * @param {string} url An absolute URL
+     * @param {string} linkName Optional name of the link
+     * @returns {string} The URL formatted as markdown link
+     */
+    _formatAsMarkdownLink = (url, linkName) => {
```

Comment:
I don't quite get why you added this function. Only this path is used
```js
return `[${url}](${url})`
```

Reply:
That you for the review and feedback.
Initially the `_formatAsMarkdownLink` function was supposed to be used more often, but in the end it doesn't make sense.

Regarding the other topics: I've incorporated your suggestions in the card layout and added the before/after screenshots.
As for the part about the expired certificates, I could find any differences except the description. Did you expect any other outcome?

If there are any other suggestions, feel free to write them, too.